### ISSUE TITLE
problem: requires a mono version that does not exist

### ISSUE THREAD

**Issue by tpokorra** [tpokorra/lbs-nant#1]
see also https://sourceforge.net/p/nant/bugs/498/

```
[00186] Error: Package: nant-0.92.999-0.x86_64 (lbs-tpokorra-nant)
[00186]            Requires: mono(System) = 1.0.3300.0
[00186]            Available: mono-core-3.12.0-0.x86_64 (download.mono-project.com_repo_centos_)
[00186]                mono(System) = 1.0.5000.0
[00186]                mono(System) = 2.0.0.0
[00186]                mono(System) = 4.0.0.0
[00186]            Installing: mono-core-3.12.0-1.x86_64 (download.mono-project.com_repo_centos_)
[00186]                mono(System) = 1.0.5000.0
[00186]                mono(System) = 2.0.0.0
[00186]                mono(System) = 4.0.0.0
```

**Comment by tpokorra** [comment]
see also http://lists.ximian.com/pipermail/mono-devel-list/2006-July/019412.html
we need >= instead of = for the requires

**Comment by tpokorra** [comment]
see http://stackoverflow.com/questions/16598201/disable-rpmbuild-automatic-requirement-finding

in spec file: AutoReqProv: no

also take a look at /usr/lib/rpm/find-requires

**Comment by tpokorra** [comment]
```
echo ../../BUILD/nant-7906a4d7e903b0ee26c466fefa58d7ba730f534c/build/mono-2.0.unix/nant-debug/bin/log4net.dll|/usr/lib/rpm/mono-find-requires
mono(mscorlib) = 1.0.3300.0
mono(System) = 1.0.3300.0
mono(System.Data) = 1.0.3300.0
mono(System.Web) = 1.0.3300.0
mono(System.Xml) = 1.0.3300.0
```

/usr/lib/rpm/mono-find-requires is a bash script, and it should contain >= instead of =

it belongs to rpm-build package:

```
rpm -qf /usr/lib/rpm/mono-find-requires
rpm-build-4.11.1-18.el7_0.x86_64
```

this shows that there is actually a fix for the problem with 1.0.3300.0 requirements:

```
diff -uNr /usr/lib/rpm/mono-find-requires /usr/bin/mono-find-requires
```
